### PR TITLE
move and rename repocheck api endpoints

### PR DESCRIPTION
**Pull request by MaximilianMeister**
```perl
/api/crowbar/repocheck -> /api/upgrade/adminrepocheck
/api/upgrade/repocheck -> /api/upgrade/noderepocheck
```

`crowbarctl` PR https://github.com/crowbar/crowbar-client/pull/114

needs a (partial) backport

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
[(3) S49P8 Move upgrade-related API endpoints to the correct namespace](https://trello.com/c/RGpe9W7m/143-3-s49p8-move-upgrade-related-api-endpoints-to-the-correct-namespace)

File: crowbar_framework/app/controllers/api/upgrade_controller.rb [hunk 1]
```diff
@@ -200,8 +200,43 @@ def cancel
...
+    "Sanity check for Crowbar server repositories"
+  header "Accept", "application/vnd.crowbar.v2.0+json", required: true
+  api_version "2.0"
+  example '
+  {
+    "os": {
+      "available": true,
+      "repos": {}
+    },
+    "cloud": {
```
Comment (jsuchome) [inline crowbar_framework/app/controllers/api/upgrade_controller.rb]: It should be "openstack" I believe

**Comment by aspiers** [comment]
The `api/` prefix makes perfect sense for the endpoints and controllers, but it makes no sense to me at all in the models.

**Comment by rsalevsky** [comment]
@aspiers Do you mean the part with the noderepocheck?

**Pull request merged**